package.json

```diff
@@ -58,11 +58,11 @@
   },
   "devDependencies": {
     "@ant-design/icons": "4.7.0",
-    "@commitlint/cli": "17.1.2",
-    "@commitlint/config-conventional": "17.1.0",
-    "@emotion/babel-plugin": "11.10.2",
-    "@emotion/react": "11.10.4",
-    "@emotion/styled": "11.10.4",
+    "@commitlint/cli": "17.2.0",
+    "@commitlint/config-conventional": "17.2.0",
```

Comment:
Please revert these changes to `package.json` and `yarn.lock` since they are not needed for this PR.